clients/java/src/main/java/io/camunda/zeebe/client/api/search/query/SearchRequestPage.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.camunda.zeebe.client.api.search;
+package io.camunda.zeebe.client.api.search.query;
```

Comment:
🔧 This file shouldn't be in this package. I would move it outside the query package and leave it into the `/search` root